### PR TITLE
Changes needed for VM gen refactoring(Pre-processing)

### PR DESCRIPTION
**Pull request by sungeunbae**
Changes made during Pre-processing vm refactoring. 
https://github.com/ucgmsim/Pre-processing/pull/179

### CODE REVIEW

**Review by jasonmotha — DISMISSED** [review]

**Review by jpa198 — DISMISSED** [review]

**Comment by sungeunbae** [comment]
Let's get this one sorted first, please?

**Comment by sungeunbae** [comment]
> Let's get this one sorted first, please?

@jasonmotha @jpa198

File: qcore/geo.py [hunk 1]
```diff
@@ -465,6 +465,58 @@ def avg_wbearing(angles):
     return degrees(atan(x / y) + q_diff)
 
 
+def build_corners(origin, rot, xlen, ylen):
+    # wanted xlen, ylen is at corners
```
Comment (jasonmotha) [inline qcore/geo.py:469]: Docstring? Specifically looking for what's the format of the return type.

**Review by jasonmotha — CHANGES_REQUESTED** [review]
looks good. Just 1 quick comment.

File: qcore/geo.py (hunk 1 above)
Comment (sungeunbae) [reply qcore/geo.py:482]: Done.

**Review by jasonmotha — APPROVED** [review]

**Review by jpa198 — APPROVED** [review]

**Pull request merged**